### PR TITLE
add sounf boss muttering when tutorial event triggers

### PR DESCRIPTION
**Pull request by Sebasgut1**
I used my voice :p

### CODE REVIEW

File: GameDesign/Assets/Scripts/Events/GameEventTrigger.cs [hunk 1]
```diff
@@ -31,6 +33,12 @@ public void TriggerEventViaCode(Collider other)
 
         if (EventTriggerCondition(other))
         {
+            Debug.LogError(PlayerPrefs.GetInt(m_event.name, 0));
```
Comment (theenergici) [inline GameDesign/Assets/Scripts/Events/GameEventTrigger.cs:36]: why is this a LogError?

File: GameDesign/Assets/Scripts/Events/GameEventTrigger.cs [hunk 2]
```diff
@@ -31,6 +33,12 @@ public void TriggerEventViaCode(Collider other)
...
+            if (!m_hasGameEventBeenTriggered && PlayerPrefs.GetInt(m_event.name, 0) != 1)
+            {
+                bossMutter = GetComponent<AudioSource>();
+                bossMutter.Play();
```
Comment (theenergici) [inline GameDesign/Assets/Scripts/Events/GameEventTrigger.cs:40]: You need to add a null check. not all events will have an audio source

File: GameDesign/Assets/Scripts/Events/GameEventTrigger.cs (hunk 1 above)
Comment (theenergici) [inline GameDesign/Assets/Scripts/Events/GameEventTrigger.cs:36]: This shouldnt be a LogError, it stops the game when you get to this point, if this code is running

**Review by theenergici — CHANGES_REQUESTED** [review]

File: GameDesign/Assets/Scripts/Events/GameEventTrigger.cs (hunk 2 above)
Comment (theenergici) [reply GameDesign/Assets/Scripts/Events/GameEventTrigger.cs:40]: also technically you should have the playing of the audio on the listener, not on the trigger

Comment (Sebasgut1) [reply GameDesign/Assets/Scripts/Events/GameEventTrigger.cs:40]: Who is the listener?

**Review by theenergici — APPROVED** [review]

**Pull request merged**